ui/raidboss/data/03-hw/raid/a8n.js

```diff
@@ -221,9 +219,7 @@ export default {
       netRegexCn: NetRegexes.ability({ source: '残暴正义号', id: ['1750', '1756'] }),
       netRegexKo: NetRegexes.ability({ source: '포악한 심판자', id: ['1750', '1756'] }),
       suppressSeconds: 5,
-      run: (data, matches) => {
-        data.bruteTankOut = matches.id === '1756';
-      },
+      run: (data, matches) => data.bruteTankOut = matches.id === '1756',
```

Comment:
This is the only `==` I found within the diff (not that there isn't other implicit ways to return)

Reply:
Do you mean `===` and not `==`? I'm not sure what you're commenting on, sorry?

Reply:
Either, `===` will be caught by searching for `==`; I'm mainly looking to see if there's any form of logic changes (although I think we don't care about the return value from preRun/run ?)